src/pages/Pool/PositionPage.tsx

```diff
@@ -654,7 +661,12 @@ export function PositionPage() {
                     marginRight: '12px',
                   }}
                 >
-                  <div style={{ marginRight: 12 }}>
+                  <div
+                    style={{
+                      width: 232,
+                      height: 400,
+                    }}
+                  >
```

Comment:
To fix this layout shift under Safari:

https://user-images.githubusercontent.com/1091472/181688010-10f0a96b-2f1c-4fb8-a698-d879ee7f05c9.mov